examples/linux_userspace.c

```diff
@@ -153,18 +153,19 @@ int main(int argc, char* argv[])
         fprintf(stderr, "Failed to open the i2c bus %s\n", argv[1]);
         exit(1);
     }
-
-#ifdef __KERNEL__
+    
+    id.dev_addr = BME280_I2C_ADDR_PRIM;
+//#ifdef __KERNEL__
     if (ioctl(id.fd, I2C_SLAVE, id.dev_addr) < 0)
     {
         fprintf(stderr, "Failed to acquire bus access and/or talk to slave.\n");
         exit(1);
     }
 
-#endif
+//#endif
 
     /* Make sure to select BME280_I2C_ADDR_PRIM or BME280_I2C_ADDR_SEC as needed */

```

Comment:
Given that you've moved the line of code that this is referring to, you probably want to move the comment too =)